examples/capture/main.rs

```diff
@@ -1,72 +1,74 @@
 /// This example shows how to capture an image by rendering it to a texture, copying the texture to
 /// a buffer, and retrieving it from the buffer. This could be used for "taking a screenshot," with
 /// the added benefit that this method doesn't require a window to be created.
+use wgpu::*;
```

Comment:
Global imports have very limited uses (prelude and tests). Let's not use them here.

Reply:
Okay, sounds good. I have switched to explicit imports, but I would also be open to only importing `wgpu` if you think that's better.

Reply:
Please follow the examples and use `wgpu` only import. If you don't think it's good, let's talk this over (on gitter, preferably) in the context of all the examples - no need to break consistency before there is any dialog on the problem.

Reply:
Okay, that makes sense. I just pushed a commit to maintain consistency with the other examples.